sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/TimeSeriesInsightsInstancesTests.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.IoT.TimeSeriesInsights.Tests
 {
     public class TimeSeriesInsightsInstancesTests : E2eTestBase
     {
-        private static readonly TimeSpan s_retryDelay = TimeSpan.FromSeconds(10);
+        private static readonly TimeSpan s_retryDelay = TimeSpan.FromSeconds(20);
```

Comment:
Did you see it failed more frequently earlier?

Reply:
I've noticed that sometimes it takes TSI a little bit longer for things to settle well in their DBs. I saw that adding the retry delay helped.